app/(dashboard)/[category]/list/filters/reactionFilter.tsx

```
@@ -8,27 +9,33 @@ import {
-export function ReactionFilter({
+export const ReactionFilter = memo(function ReactionFilter({
   reactionType,
   onChange,
 }: {
   reactionType: "thumbs-up" | "thumbs-down" | null;
   onChange: (reactionType: "thumbs-up" | "thumbs-down" | null) => void;
 }) {
+  const buttonContent = useMemo(() => {
+    const icon = reactionType === "thumbs-down" 
+      ? <ThumbsDown className="h-4 w-4 mr-2" />
+      : <ThumbsUp className="h-4 w-4 mr-2" />;
```

ESLint

[error] Replace `·reactionType·===·"thumbs-down"·⏎······?·<ThumbsDown·className="h-4·w-4·mr-2"·/>⏎·····` with `⏎······reactionType·===·"thumbs-down"·?·<ThumbsDown·className="h-4·w-4·mr-2"·/>`

(prettier/prettier)